float32[4] delta_x
float32[4] delta_y
float32[4] delta_z
float32[4] delta_yaw
float32 team_pos_x
float32 team_pos_y
float32 team_pos_z
float32 team_pos_yaw